.github/workflows/test_tokio.yml

```diff
@@ -0,0 +1,140 @@
+on: [push, pull_request]
+
+# on:
+#   push:
+#     branches: ["master"]
+#   pull_request:
+#     branches: ["master"]
+
+name: Test tokio
+
+env:
+  RUSTFLAGS: -Dwarnings
+  nightly: nightly-2020-01-25
+
+jobs:
+  test_tokio:
+    name: Test tokio full
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-latest
+          - macos-latest
+    steps:
+      - uses: actions/checkout@master
+
+      # Run `tokio` with only `full`
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --manifest-path ${{ github.workspace }}/tokio/Cargo.toml --features full
+        env:
+          RUST_BACKTRACE: 1
```

Comment:
nit: probably we can move this into [the top-level env field](https://github.com/tokio-rs/tokio/pull/2531/files#diff-d3c6bb574b84b525c94de18f2617a66dR11-R13)